Server/API/rooms/guess/postGuess.go

```diff
@@ -0,0 +1,51 @@
+package guess
+
+import (
+	"context"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/labstack/echo/v4"
+	roomsScripts "ishtaloo.io/DB/Scripts/rooms"
+	cookieUtils "ishtaloo.io/Utils/cookie"
+)
+
+func postGuess(c echo.Context) (err error) {
+	roomId := c.Param("roomId")
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	u, _ := cookieUtils.GetUserFromCookie(c)
+	usersInRoom, err := roomsScripts.GetUsersOfRoom(ctx, roomId)
+	if err != nil {
+		return c.String(http.StatusInternalServerError, strings.Join([]string{err.Error(), "from GetUserOfRoom"}, ", "))
```

Comment:
I see many methods to return an error, can you create like a Util file like handleInternalServerError(err, msg?) or HandleForbidden(err, msg?), I think this will tidy up the code quite a bit

Reply:
Really? I don't think it will add real readability